core/src/main/java/tachyon/thrift/ClientFileInfo.java

```diff
@@ -610,7 +610,7 @@ public void addToBlockIds(long elem) {
   }
 
   public List<Long> getBlockIds() {
-    return this.blockIds;
+    return this.blockIds == null ? new ArrayList<Long>() : this.blockIds;
```

Comment:
Is this the only way to do this? Since it will get overridden whenever someone does thriftGen making it difficult to maintain right?